rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+
+New items in the *dcp* repo appear in the ZenHub *New* pipeline and MUST be triaged into the *Product Backlog* pipeline, the *Icebox* pipeline, or the *Closed* pipeline by the Release Shepherd or Product Owners.
+
+The ZenHub *Icebox* pipeline SHOULD be used with great restraint in the *dcp* repo. Its explicit purpose is to queue up potential futures from the DCP roadmap such as pending DevSecOps issues. Individual component repo(s)  MAY use the *Icebox* pipeline in a manner of their own choosing.
+
+No issues in the *dcp* repo MUST be assigned to the *Epic* or *Sprint Backlog* pipelines. Individual component repo(s) MAY use these pipelines in a manner of their own choosing.
+
+At the level of the *dcp* repo, a subset of ZenHub pipelines are used to track progress. All Epics or issues in a Release are initially assigned to the *Product Backlog* pipeline. Once the Epic (or any of its children) is started, then its *dcp* issue is moved to the *In Progress* pipeline. When all the children of the Epic are in the *Closed* pipeline, then the *dcp* issue is moved to the *Closed* pipeline.
+
+#### Modeling progress with ZenHub Pipelines [component repo]
+
+When an issue is started, it is moved to the *In Progress* pipeline.
+
+When a pull request is filed and in review for an *In Progress* item, the issue is moved to the *Review* pipeline.
+
+If the pull request is rejected, the issue returns to *In Progress*; otherwise, it is merged and the related issue moves to the *Merged* Pipeline.
+
+When all changes are verified in [Production](https://allspark.dev.data.humancellatlas.org/dcp-ops/docs/wikis/SOP:%20Releasing%20new%20Versions%20of%20DCP%20Software), the issue is moved to the *Closed* pipeline.
+
+### Release Demonstration and Retrospective
+
+**At Release Completion**. The Release Shepherd MUST schedule and faciliate a Release Demonstration and Retrospective meeting with the DCP Community and Oversight. It is RECOMMENDED that this occur during a DCP F2F when schedules align. 
+
+The intent of the meeting is to demonstrate the Release deliverables as documented in the Roadmap Objectives, to share learnings, and identify opportunities to improve the planning process for future Releases. 
+
+### Acceptance Criteria
+
+This process will be under continuous community introspection and improvement through Retrospectives. If the process fails to deliver product increments with high confidence after three quarters with an annual Roadmap, then more radical alternatives SHOULD be pursued that satisfy the requirements from Oversight. We SHOULD also assess whether ZenHub is meeting our needs for a common project management application. 
+
+### Unresolved Questions
+
+*What aspects of the design do you expect to clarify further through the RFC review process?*
+
+The authors are seeking recommendations for:
+* Improving the accountability for Release schedule and deliverables.
+* Assigning owners to Release Objectives with appropriate load balancing
```

Comment:
It is intriguing that you pose this a question here because one of my general comments was going to be that it seems like with this process it becomes easy to create a lot of siloed work and the cross-cutting work that actually involves every team (much like some of the metadata work this quarter) won't have a natural PO to be assigned to. How would you recommend executing/creating ownership of these types of themes/objectives?

Reply:
Because the roadmap objectives focus on end-to-end user stories (epics), it's actually harder to create _siloed work_.  The Project Leads struggled with a similar issue when considering how to assign a Theme _Accountable_ in their RACI but arrived at no general solution to my knowledge. The one constraint that I've considered is that the Product Owner who is assigned to an objective SHOULD have a deliverable at part of that objective.